### DISCUSSION TITLE
Glide slope and range OSD indicators

### DISCUSSION THREAD

**Discussion by Yury-MonZon** [iNavFlight/inav#10112] [General]
In my understanding it should be showing values only when there is no throttle. 

Is there any reason to have it when flying with throttle?

**Comment by MrD-RC** [comment]
Glide slope is a term for the decent path. Airliners on approach are on a glide slope. So it is not related to throttle at all. You could be under power, and still descending on a glide slope.

**Reply by Yury-MonZon** [reply]
Understood, thanks.
